.github/workflows/pr-check.yaml

```diff
@@ -13,12 +13,9 @@ jobs:
         with:
           node-version: '20'
       - name: Install AsyncAPI Generator
-        # The @asyncapi/generator version is pinned to the latest version that's
-        # compatible with the current latest version of @asyncapi/html-template
-        # (v2.3.5).
-        run: npm install -g @asyncapi/generator@1.17.25
+        run: npm install -g @asyncapi/cli
```

Comment:
It doesn't look like we need to pin the version of `@asyncapi/generator` anymore. When I tried using the latest version locally, it worked fine.